src/averageAges.js

```diff
@@ -1,61 +1,100 @@
 'use strict';
 
 /**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
  * @param {object[]} people
  * @param {number} century - optional
  *
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let menArray = people.filter((man) => {
+    return man.sex === 'm';
+  });
+
+  if (century) {
```

Comment:
Please check comment to this task
![image](https://github.com/mate-academy/js_average-ages/assets/54936044/5051205d-5ff1-40dc-bb4b-e21d27c2d2ce)